src/v2/modules/documents/validator.js

```diff
@@ -43,11 +42,23 @@ const documentRoleSchema = Joi.object({
  * Validates that an object conforms to the requirements of a document role
  * based on the role being proposed.
  */
-exports.validateDocumentRole = documentRole =>
-  Joi.validate(documentRole, documentRoleSchema, { abortEarly: false });
+exports.validateDocumentRole = documentRole => {
+  const valRes = documentRoleSchema.validate(documentRole, { abortEarly: false });
+  const { error, value } = valRes;
+  if (error) {
+    return { value, error };
+  }
+  return { value, error: null };
+};
 
 /**
  * Validates that an object conforms to the requirements of an document.
  */
-exports.validateDocument = document =>
-  Joi.validate(document, documentSchema, { abortEarly: false });
+exports.validateDocument = document => {
+  const valRes = documentSchema.validate(document, { abortEarly: false });
```

Comment:
you should be able to just return from this line i.e. 
`return documentSchema.validate(document, { abortEarly: false });`